src/bun.js/javascript.zig

```diff
@@ -3616,13 +3607,24 @@ pub const VirtualMachine = struct {
                 .data => |data| {
                     IPC.log("Received IPC message from parent", .{});
                     if (this.globalThis) |global| {
+                        if (data.isObject()) {
+                            if (data.get(global, "cmd")) |prop| {
```

Comment:
This is an expensive function call inside hot code. Do we need to do this? Can we instead make it a separate message type in the encoding?